docs/source/_sidebar.yaml

```
@@ -0,0 +1,293 @@
+            href: "./performance/caching/in-memory"
+          - label: "Distributed Caching"
+            href: "./performance/caching/distributed"
+          - label: "Entity Caching"
+            href: "./performance/caching/entity"
+      - label: "Performance and Scaling"
+        children:
+          - label: "Overview"
+            href: "./performance"
+          - label: "Traffic Shaping"
+            href: "./performance/traffic-shaping"
+          - label: "Query Batching"
+            href: "./performance/query-batching"
+      - label: "Client Features"
+        children:
+          - label: "HTTP Header Propagation"
+            href: "./header-propagation"
+          - label: "@defer"
+            href: "./operations/defer"
+          - label: "GraphQL Subscriptions"
+            children:
+              - label: "Overview"
+                href: "./operations/subscriptions/overview"
+              - label: "Configuration"
+                href: "./operations/subscriptions/configuration"
+              - label: "Callback Protocol"
+                href: "./operations/subscriptions/callback-protocol"
+              - label: "Multipart Protocol"
+                href: "./operations/subscriptions/multipart-protocol"
+              - label: "Enable with API Gateway"
+                href: "./operations/subscriptions/api-gateway"
+          - label: "File Upload"
+            href: "./operations/file-upload"
+      - label: "Customization"
+        children:
+          - label: "Overview"
+            href: "./customization/overview"
+          - label: "Coprocessors"
+            children:
+              - label: "Coprocessor Configuration"
+                href: "./customization/coprocessor"
+              - label: "Coprocessor Reference"
+                href: "./customization/coprocessor/reference"
+          - label: "Rhai Scripts"
+            children:
+              - label: "Rhai Configuration"
+                href: "./customization/rhai"
+              - label: "Rhai API Reference"
+                href: "./customization/rhai/reference"
+          - label: "Custom Builds"
+            children:
+            - label: "Building a Binary"
+              href: "./customization/custom-binary"
+            - label: "Rust Plugins"
+              href: "./customization/native-plugins"
+  - label: "Deployment"
+    children:
+      - label: "Overview"
+        href: "./self-hosted"
+      - label: "Docker"
+        href: "./self-hosted/containerization/docker"
+      - label: "Kubernetes"
+        children:
+          - label: "Quickstart"
+            href: "./self-hosted/kubernetes/quickstart"
+          - label: "Deploying with extensibility"
+            href: "./self-hosted/kubernetes/extensibility"
+          - label: "Enabling metrics"
+            href: "./self-hosted/kubernetes/metrics"
+          - label: "Other consideration"
+            href: "./self-hosted/kubernetes/other-considerations"
```

Missing the `containerization` dir in the path:

```suggestion
            href: "./self-hosted/containerization/kubernetes/quickstart"
          - label: "Deploying with extensibility"
            href: "./self-hosted/containerization/kubernetes/extensibility"
          - label: "Enabling metrics"
            href: "./self-hosted/containerization/kubernetes/metrics"
          - label: "Other consideration"
            href: "./self-hosted/containerization/kubernetes/other-considerations"
```